Testing/Unit/Rego/Defender/DefenderConfig_01_test.rego

```diff
@@ -1,185 +1,171 @@
-package defender
+package defender_test
```

Comment:
I assume the package rename in general was to reduce collisions in namespace.  Is it worth further partitioning?  For example, `package defender_test_01`

Reply:
This is because of the rules in our linter. We decided to add `test cannot be outside test package` rule, so all files with test must be in a package ending with `_test`